src/node/markdown/plugins/highlight.ts

```diff
@@ -1,14 +1,28 @@
-import escapeHtml from 'escape-html'
 import { getHighlighter } from 'shiki'
+import type { ThemeOptions } from '../markdown'
 
-export const highlight = async (theme = 'material-palenight') => {
-  const highlighter = await getHighlighter({ theme })
+export async function highlight(
+  theme: ThemeOptions = { light: 'vitesse-light', dark: 'vitesse-dark' }
```

Comment:
By no means to self prompt :P 

Was trying to demonstrate this feature, but `material-palenight` does not seem to have a light mode pair.

Feel free to change the themes that make sense to you: https://github.com/shikijs/shiki/tree/main/packages/shiki/themes